rag-sql-router/app.py

```
@@ -0,0 +1,454 @@
+torch.classes.__path__ = []
+
+import os
+import uuid
+import random
+import time
+import tempfile
+import streamlit as st
+
+# Import necessary components from llama_index
+from llama_index.core import Settings
+from llama_index.llms.openai import OpenAI
```

Ruff (0.12.2)

`llama_index.llms.openai.OpenAI` imported but unused

Remove unused import: `llama_index.llms.openai.OpenAI`

(F401)